### ISSUE TITLE
Points are not parsed in xaml if using a space for point separator

### ISSUE THREAD

**Issue by donandren** [AvaloniaUI/Avalonia#3813]
Xaml like:
`<Polygon xmlns="https://github.com/avaloniaui" Points="10 20 30 40" />`
is producing exception - `System.FormatException`

Wpf parses this without error.

Note: if xaml is: `<Polygon xmlns="https://github.com/avaloniaui" Points="10,20 30,40" />`
it's working

**Comment by derekantrican** [comment]
I'm getting the same error (System.FormatException - Invalid point) with this:

```xml
<Polygon Points="0,0 80,50, 0,100" Stroke="Black" Fill="Black" />
```

But @donandren's second example works for me. Do I need to change my formatting?

**Comment by derekantrican** [comment]
Weird - if I copy and paste that line it throws the exception on startup. But If I delete all the points and type them in again manually it works. Any reason why it wouldn't be able to process the points when copy & pasted?

**Comment by maxkatz6** [comment]
> I'm getting the same error

PR #3814 was merged to 0.10, I don't think it was backported to 0.9.
